php/classes/controllers/class-players-controller.php

```diff
@@ -65,17 +65,20 @@ public function get_latest_episode_id() {
 	 *
 	 * @param int $id
 	 *
-	 * @return string
+	 * @return array
 	 */
 	public function html_player( $id ) {
+		if ( empty( $audio_file = get_post_meta( $id, 'audio_file', true ) ) ) {
+			return apply_filters( 'ssp_html_player_data', [] );
```

Comment:
Following WordPress Coding Standards, please declare arrays using the long array syntax, in the plugin.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#declaring-arrays

Reply:
Done